ui-participant/src/landing/sections/LinkSectionsFooter.tsx

```diff
@@ -55,7 +55,7 @@ const getIcon = (item: ButtonConfig) => {
 type LinkSectionsFooterProps = TemplateComponentProps<LinkSectionsFooterConfig>
 
 /** renders a footer-style section */
-export function LinkSectionsFooter(props: LinkSectionsFooterProps) {
+function LinkSectionsFooter(props: LinkSectionsFooterProps) {
```

Comment:
ESLint warns about possibly confusing
```ts
import LinkSectionsFooter from 'path/to/LinkSectionsFooter'
```
and
```ts
import { LinkSectionsFooter } from 'path/to/LinkSectionsFooter'
```

Removing this unused export satisfies it.